Sources/TuistLoader/Loaders/ManifestModelConverter.swift

```diff
@@ -5,8 +5,43 @@ import TuistCore
 import TuistGraph
 import TuistSupport
 
-/// A component responsible for converting Manifests (`ProjectDescription`) to Models (`TuistCore`)
```

Comment:
why did you remove it? 🤔

Reply:
Ah sorry I lost it in the file move - will restore it